src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_acl.erl

```diff
@@ -254,7 +254,12 @@ names_to_ids(ACE, OrgId) ->
             safe_fetch_ids(user, undefined, ej:get({<<"users">>}, ACE)) ++
             safe_fetch_ids(client, OrgId, Clients)
     end,
-    GroupIds = safe_fetch_ids(group, OrgId, ej:get({<<"groups">>}, ACE)),
+    %% Global groups can be intermixed with org local groups. This creates some ambiguity as
```

Comment:
I think this is the argument for moving server-admins to a globally distinct namespace; a reserved prefix would help us distinguish the two.